core/frontend/src/store/notifications.ts

```diff
@@ -0,0 +1,53 @@
+import { Module, VuexModule, Mutation } from 'vuex-module-decorators'
+import { CumulatedNotification, Notification } from '@/types/notifications'
+import store from '@/store'
+
+
+@Module({
+  dynamic: true,
+  store,
+  name: 'notification_store',
+})
+
+@Module
+export default class NotificationStore extends VuexModule {
+  notifications: Notification[] = []
+  cumulated_notifications: CumulatedNotification[] = []
+  dismissed_notifications_hashes: string[] = []
+
+  @Mutation
+  pushNotification (notification: Notification): void {
+    this.notifications.push(notification)
```

Comment:
Can we use console here to have also the notification/message log in the browser console and not only in the notification web ui ?

Reply:
Will add.